src/merge.ts

```diff
@@ -10,7 +10,7 @@ import { EventPayload } from './EventPayload';
  *  Merging a non-existing key adds the key/value.
  *  When the value is an object, the above is applied iteratively.
  */
-export const merge = (original: EventPayload, newValues: PartialPayload): EventPayload => {
+const merge = (original: PartialPayload, newValues: PartialPayload): EventPayload => {
   const copy = JSON.parse(JSON.stringify(original));
```

Comment:
Added a check for that at the top of merge. 

However in the report function I believe we would still need to check if original is undefined, since right now the Reporter is setup so that the payload object (in the constructor) is optional.